components/ntask/org.wso2.carbon.ntask.core/src/main/java/org/wso2/carbon/ntask/core/internal/TasksDSComponent.java

```diff
@@ -119,6 +120,29 @@ private Properties getStandardQuartzProps() {
         return result;
     }
 
+    /**
+     * Method to retrieve default Quartz properties when clustering is enabled.
+     * Points to a datasource configured in master-datasources.xml with JNDI name QuartzDB.
+     *
+     * @return Quartz properties
+     */
+    private Properties getClusteredQuartzProps() {
+        if (log.isDebugEnabled()) {
+            log.debug("No user specified quartz properties found for clustered mode: using default values.");
+        }
+        Properties result = new Properties();
+        result.put("org.quartz.scheduler.instanceName", "MyScheduler");
+        result.put("org.quartz.scheduler.instanceId", "AUTO");
+        result.put("org.quartz.threadPool.class", QuartzCachedThreadPool.class.getName());
+        result.put("org.quartz.jobStore.class", JobStoreTX.class.getName());
+        result.put("org.quartz.jobStore.driverDelegateClass", StdJDBCDelegate.class.getName());
+        result.put("org.quartz.jobStore.dataSource", "quartzDS");
+        result.put("org.quartz.jobStore.isClustered", "true");
+        result.put("org.quartz.jobStore.clusterCheckinInterval", "5000");
+        result.put("org.quartz.dataSource.quartzDS.jndiURL", "QuartzDB");
+        return result;
```

Comment:
We have not set the org.quartz.scheduler.skipUpdateCheck value to true in clustered properties. Better to have a private method with common properties and use it in both standalone and clustering methods.

Reply:
Done. (Introduced new method - getCommonQuartzProps()).